ClinGen/clincoded
# Connects to #1695. Connects to #1696. Connects to #1697. Connects to #1699. Connects to #1702. Connects to #1704.

**Steps to test #1695:**
1. Go to the **VCI** and enter **ClinVar ID 92494** in the variant lookup modal.
2. Proceed to the **Population** tab in the evidence view.
3. In the **ExAC** table, expect to see the **Total** number being shown for the **Number of Homozygotes** column.

***

**Steps to test #1696 (not applicable to production instances):**
1. On a demo/test instance, login with your stanford.edu user account.
2. Upon successfully login, expect to see the affiliation modal being shown.
3. Click on the **Select Affiliation** dropdown, expect to see the **ClinGen Curator Test Affiliation 1** and **ClinGen Curator Test Affiliation 2** option being shown.
4. Select the **ClinGen Curator Test Affiliation 1** affiliation and proceed with curating/scoring some evidence in a GDM. Continue to save/provision/approve the **Classification** for the GDM.
5. In the **Approve Classification** panel, click on the **Approver** dropdown. Expect to see the **Person A** and **Person B** options being shown.

***

**Steps to test #1697:**
1. On a demo/test instance, login with your stanford.edu user account.
2. Assign yourself to all the current affiliations (10001 through 10026).
3. Logout and login again. In the affiliation modal, click on the **Select Affiliation** dropdown. expect to see the shown affiliation names now ending with **EP** instead of the  previous **Expert Panel**, as well as the removal of the **ClinGen** prefix.
4. Refer to https://github.com/ClinGen/clincoded/issues/1697#issuecomment-398762375 for cross-checking the correct affiliation names.

***

**Steps to test #1699:**
1. Create a new **Family** evidence with a **Proband** added to it.
2. Before clicking on the **Save** button, open the **Chrome Developer Tools**.
![image](https://user-images.githubusercontent.com/6956310/41570809-2e8715da-7325-11e8-869b-c7a2216ad726.png)
3. In the **Chrome Developer Tools**, select the **Network** tab: 
![image](https://user-images.githubusercontent.com/6956310/41570863-613e400c-7325-11e8-9888-5f3063bb7d36.png)
4. Then select the **XHR** subtab:
![image](https://user-images.githubusercontent.com/6956310/41570894-884fdcb4-7325-11e8-8fba-04c2502ee934.png)
5. Click on the **Clear** icon to remove all the existing logged requests:
![image](https://user-images.githubusercontent.com/6956310/41570929-b09c29e8-7325-11e8-8638-fb527a8412d7.png)
6. Now click on the **Save** button at the bottom of the **Family** evidence.
7. In the **Network** subtab of the **Chrome Developer Tools**, click on the **individuals/** request:
![image](https://user-images.githubusercontent.com/6956310/41571095-6db725b4-7326-11e8-8f9d-41903fe585b4.png)
8. In the request inspector panel, scroll to the bottom where you will see the expandable **Request Payload**.
9. Expand it and the subsequent _JSON_ object. Find the **method** nested object and expand it as well. Expect to see the **dateTime** field with a timestamp string value.

***

**Steps to test #1702:**
1. In the **GCI**, curate/score some evidence in a GDM.
2. Click on the **Classification Matrix** button at the upper-right.
3. On the subsequent matrix page, expect to see the **fuchsia** placeholder text in the **Explain Reason(s) for Change** and **Evidence Summary (optional)** text fields.

***

**Steps to test #1704:**
1. Go to the **VCI** and enter **CA IDs** either **CA658820601** or **CA658820611** in the variant lookup modal.
2. Expect to see variant data being retrieved and shown in the modal without error.
3. Click on the **Save** button in the modal and expect to be able to proceed to the **VCI** evidence view.
